com.microsoft.mrtk.accessibility/Utilities/DescribableObject.cs

```diff
@@ -97,6 +77,19 @@ public DescribableFlag Flags
         /// </remarks>
         public string Instructions => instructions;
 
+        [SerializeField]
+        [Tooltip("Is the object salient to the current state of the experience?")]
+        private bool isSalient = true;
+
+        /// <summary>
+        /// Is the object salient to the current state of the experience?
+        /// </summary>
+        public bool IsSalient
```

Comment:
I think this is super nitty, but could we use like...`IsRelevant` or `IsContextuallyRelevant` or something here? Basically, to use (imo) more common verbiage.

Reply:
sure. i like IsContextuallyRelevant

Reply:
i'll make that change. you might need to re-review afterward :smile: